app/helpers/placements/support/organisations_helper.rb

```diff
@@ -11,6 +11,15 @@ def no_results(search = "", filters = [])
     end
   end
 
+  def organisation_url(organisation)
+    case organisation.class.name
+    when "School"
+      placements_support_school_path(organisation)
+    when "Provider"
+      placements_support_provider_path(organisation)
+    end
+  end
```

Comment:
This won't work for `Placements::School` models, as you're comparing the class names as strings, not the actual classes/instances.